ar_exercise/exercise.rb

```diff
@@ -32,17 +32,38 @@ class Apartment < ActiveRecord::Base
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
 all_tenants = Tenant.all
-
 # get the first tenant in the DB
+first_tenants = Tenant.first
 # get all tenants older than 65
+old_tenants = Tenant.find_by(age: >65)
```

Comment:
If you want to incorporate a conditional into an Active Record method, you will need to use different syntax. Take a look at this Stackoverflow post: https://stackoverflow.com/questions/11317662/rails-using-greater-than-less-than-with-a-where-statement